src/tim/vx/platform/grpc/remote_service.cmake

```diff
@@ -0,0 +1,59 @@
+list(APPEND ${TARGET_NAME}_SRCS
```

Comment:
why not use CMakeLists.txt ?

Reply:
It only contains some configurations about gRPC, and has no target to build. So I think **include()** is better.